docs/api.md

```diff
@@ -342,11 +344,13 @@ positiveNumber.verify('not a number');  // throws
 <a href="#positiveInteger">#</a> **positiveInteger**: <i style="color: #267f99"><a href="/Decoder.html" style="color: inherit">Decoder</a>&lt;number&gt;</i> [<small>(source)</small>](https://github.com/nvie/decoders/tree/main/src/lib/numbers.js#L42-L48 'Source')
 {: #positiveInteger .signature}
 
-Accepts only positive finite whole numbers.
+Accepts only non-negative (zero or positive) finite whole numbers.
 
 ```typescript
 // 👍
 positiveInteger.verify(123) === 123;
+positiveInteger.verify(0) === 0;
+positiveInteger.verify(-0) === -0;
```

Comment:
Philosophically speaking, this is making me very uncomfortable 🤔 While `-0 >= 0`, the negative sign really goes against the `positiveInteger` nature… 🤔 

So now I'm contemplating whether to change this implementation so it won't accept `-0` specifically.

Reply:
I decided to change the returned value to `0` in both cases when using these decoders: https://github.com/nvie/decoders/commit/d0739fd5a57a5b27ee78e81b1b353571dac4db09 — thanks for bringing this edge case to my attention!